projects/fal/src/fal/apps.py

```diff
@@ -186,7 +193,7 @@ def _connect(app_id: str, *, path: str = "/realtime") -> Iterator[_RealtimeConne
 
     from websockets.sync import client
 
-    url = _REALTIME_URL_FORMAT.format(app_id=app_id) + path
+    url = url_format(app_id, _REALTIME_URL_FORMAT) + path
```

Comment:
why not rewrite the app_id here directly so all the remaining code can stay the same?

Reply:
I don't want to inline

```py
    if "/" not in app_id:
        # Convert the app_id to the format used in the URL.
        app_id = app_id.replace("-", "/", 1)
```

on every format call I make

Reply:
no, i mean just change `app_id` here. then pass that down. we don't need to rewrite app_id everywehre

Reply:
not really sure, check the latest change?